### ISSUE TITLE
Intergrate TDI function cannot add baseline points

### ISSUE THREAD

**Issue by wbb12** [MDSplus/mdsplus#2667]
**Affiliation**
Tsinghua University

**Version(s) Affected**
7.132.0 amd64

**Platform**
Ubuntu 22.04   5.15.0-83-generic

**Describe the bug**
The integrate tdi function cannot add the baseline points argument. The function can work when only the signal expression argument is added.  However, when the base point is added to the expression, the jtraveser returns the misspelled error. 

The source code of integrate function shows that the base point argument is still supported as optional.

**Screenshots**
![img_v3_026d_f8b13104-e4d0-4715-b1a2-20c5a5e49b7g](https://github.com/MDSplus/mdsplus/assets/7125722/84f0f25f-24c6-4efb-a031-7b69a63acc08)
![img_v3_026d_f8b13104-e4d0-4715-b1a2-20c5a5e49b7g](https://github.com/MDSplus/mdsplus/assets/7125722/1f4f9fc0-96f8-420a-b9c4-039a3b0dbbd2)
![img_v3_026d_99458c1d-c4b8-4992-a604-bc5e70c6402g](https://github.com/MDSplus/mdsplus/assets/7125722/f704ac3b-b072-4d90-81dd-1d772d89905e)
<img width="848" alt="image" src="https://github.com/MDSplus/mdsplus/assets/7125722/c9442cbf-065e-460a-af31-afdf0decbe27">

**Comment by joshStillerman** [comment]
The problem is with the line:
```
                _sum = sum(_y[0.._points-1]);
```
changing it to:
```
                _sum = sum(_y[0 .. _points-1]);
```
(adding spaces around the `..` fixes it).

Now the question is:

  is `n..m` syntactically valid?  Did it used to be?  Should it be?

In addition: `baseline.fun` contains a repeated line (which should have not bad effect).

A quick fix is to add the spaces.  The parsing of `n..m` should be looked into

this is discussed as a limitation in TdiLex.l source file:
```
Limitations:
  Floating requires
    (1) digit before exponent
    (2) either decimal point or exponent, (.E3) looks like path.
    (3) NEED to handle 6..8 and 6...8 and 6...8.9, should use spaces.
    (4) 6..8 is 6 .. 8 and not 6. .8, 6...8 is ambiguous?
    (5) prefix + or - on numbers handled elsewhere, a-6 would be tokens a -6.
```

**Comment by mwinkel-dev** [comment]
Fixed by PR #2669, so closing this issue.
